quiz.js

```diff
@@ -0,0 +1,25 @@
+function populatePage (inventory) {
+	var pageInput = document.getElementById("inventory");
+	var data= "";
+
+	for(i=0; i<inventory.length; i++) {
+		data+= `<section class='col-md-4 ${i} ${inventory[i].color}'>
+						<h2>${inventory[i].make} ${inventory[i].model}</h2>
+						<p>	Year: ${inventory[i].year}
+								Price: ${inventory[i].price}
+								Color: ${inventory[i].color}
+						</p>
+						<p>Purchased: ${inventory[i].purchased}</p>
+						<p id='${i}' class='description'>${inventory[i].description}</p></section>`;
+					}
```

Comment:
Indentation is a bit weird, I'd step back lines 7-14 so that this closing ```}``` lines up with its opening brace.